src/tools/ceph_monstore_tool.cc

```diff
@@ -657,60 +657,6 @@ static int update_paxos(MonitorDBStore& st)
   return 0;
 }
 
-// rebuild
```

Comment:
could remove following references of "pgmap_meta" also:

https://github.com/ceph/ceph/blob/c98bbf4f71156c97e0bb4dbc1c5dd8f161105c21/src/tools/ceph_monstore_tool.cc#L514

and 

https://github.com/ceph/ceph/blob/c98bbf4f71156c97e0bb4dbc1c5dd8f161105c21/src/tools/ceph_monstore_tool.cc#L638

the same applies to "pgmap" and "pgmap_pg", as nobody will be reading from it anymore.

see https://github.com/ceph/ceph/blob/c98bbf4f71156c97e0bb4dbc1c5dd8f161105c21/src/tools/rebuild_mondb.cc#L15, which a part of COT.

but probably we should keep dumping `pgmap_pg/${pgid}` in `ceph-objectstore-tool`. as we can enumerate all `pgmap_pg/${pgid}` keys in `ceph-monstore-tool`, and remove the pools who don't have any PG in `pgmap_pg/${pgid}` from `creating.created_pools` .

Reply:
I'm wondering if it's better to avoid that complexity.  (1) it'd be super rare for this to happen during a pool create, and (2) If none of the pgs for the pool were created, they can either force-create-pg each pg, or delete and recreate the pool.

Reply:
(3) if none of the pgs were found, i would perhaps be nervous about implicitly recreating the pool since it might suggest there is some other bug/issue.

Reply:
then we can completely remove the `pgmap*` stuff in ceph-monstore-tool.